modules/graph_ql/output.tf

```diff
@@ -0,0 +1,4 @@
+output "graphql_load_balancer_dns_name" {
+  value       = data.aws_lb.gql_app_lb_data.dns_name
+  description = "The DNS Name of the Application Load Balancer that handles access to ORCA GraphQL."
+}
```

Comment:
Felt it was important to show how to output this, but didn't want to bubble it up to a customer-visible layer. Could change/remove this if desired.

Reply:
I feel this is good to keep as we may want to leverage this later for some internal prototyping.